plasmapy/dispersion/numerical/tests/test_hollweg_.py

```diff
@@ -119,6 +116,83 @@ def test_warning(self, kwargs, _warning):
         with pytest.warns(_warning):
             hollweg(**kwargs)
 
+    @pytest.mark.parametrize(
+        "kwargs, expected",
+        [
+            (  # beta = 1/20 for kx*L = 0
+                {**_kwargs_hollweg1999, "k": 1e-14 * u.rad / u.m, "B": 6.971e-8 * u.T},
+                1 + 0j,
+            ),
+            (  # beta = 1/20 for kx*L = 1
+                {
+                    **_kwargs_hollweg1999,
+                    "k": 0.0000439223874624874 * u.rad / u.m,
+                    "B": 6.971e-8 * u.T,
+                },
+                1.4018 + 0j,
+            ),
+            (  # beta = 1/2 for kx*L = 0
+                {**_kwargs_hollweg1999, "k": 1e-14 * u.rad / u.m, "B": 2.205e-8 * u.T},
+                1 + 0j,
+            ),
+            (  # beta = 1/2 for kx*L = 1
+                {
+                    **_kwargs_hollweg1999,
+                    "k": 0.000013893109303101 * u.rad / u.m,
+                    "B": 2.205e-8 * u.T,
+                },
+                1.3536 + 0j,
+            ),
+            (  # beta = 2 for kx*L = 0
+                {**_kwargs_hollweg1999, "k": 1e-14 * u.rad / u.m, "B": 1.097e-8 * u.T},
+                1 + 0j,
+            ),
+            (  # beta = 2 for kx*L = 1
+                {
+                    **_kwargs_hollweg1999,
+                    "k": 0.00000691190063354451 * u.rad / u.m,
+                    "B": 1.097e-8 * u.T,
+                },
+                1.2607 + 0j,
+            ),
+            (
+                {
+                    **_kwargs_hollweg1999,
+                    "k": 1e-14 * u.rad / u.m,
+                    "B": 6.97178e-7 * u.T,
+                },
+                1 + 0j,
+            ),
+            (  # beta = 1/2000 for kx*L = 1
+                {
+                    **_kwargs_hollweg1999,
+                    "k": 0.000439273010336778 * u.rad / u.m,
+                    "B": 6.97178e-7 * u.T,
+                },
+                0.98750 + 0j,
+            ),
+        ],
+    )
+    def test_hollweg1999_vals(self, kwargs, expected):
+        """
+        Test calculated values based on Figure 2 of Hollweg1999
+        (DOI: https://doi.org/10.1029/1998JA900132).
+        """
+        # k values need to be single valued for this test to function correctly
+
+        cs = pfp.cs_(kwargs["T_e"], kwargs["T_i"], kwargs["ion"]).value
+        va = pfp.va_(kwargs["B"], kwargs["n_i"], ion=kwargs["ion"]).value
+        wci = pfp.wc_(kwargs["B"], kwargs["ion"]).value
+
+        L = cs / np.abs(wci)
+        kx = (np.sin(kwargs["theta"]) * kwargs["k"]).value
+        kz = (np.cos(kwargs["theta"]) * kwargs["k"]).value
+
+        w_alfven = (hollweg(**kwargs)["alfven_mode"]).value
+        big_omega = np.abs(w_alfven / (kz * va))
+
+        assert np.isclose(big_omega, expected, atol=1e-2)
```

Comment:
I swear half of my maintainer life is just replacing `assert np.isclose` with
```suggestion
        np.testing.assert_allclose(big_omega, expected, rtol=1e-2)
```
in every test for its superior debugging output, lol. 😅 

I also took the liberty of replacing `atol` with `rtol` here. Without domain knowledge, I don't know whether an absolute difference of 0.01 is big or small here - but a difference of 1% I know we can probably live with, especially for data pulled off a plot.

Reply:
I will definitely keep the `np.testing.assert_allclose` change in mind for future tests. I'm unsure how to tell if `rtol` or `atol` would be better to use in this case. I'll wait for @rocco8773 Erik's thoughts on the matter before committing this change. 
Thank you for the help Dominik!